From 9868aca66ad424c3e7d6ae920e0eed55227c95fb Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <contact@chrisaw.io>
Date: Tue, 29 Sep 2026 13:40:40 +0100
Subject: [PATCH 2/2] disable the pvmfw test image modules on the gsi

the pvmfw test image packages protected vm firmware artifacts built
from the virtualization sources, which are excluded from the gsi
manifest sync.

Change-Id: I84af714f4a2167805a9db5e477d0225a54b09107
---
 pvmfw/Android.bp | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/pvmfw/Android.bp b/pvmfw/Android.bp
index bea7411..e2c3bda 100644
--- a/pvmfw/Android.bp
+++ b/pvmfw/Android.bp
@@ -15,7 +15,8 @@ cc_genrule {
                 ":test_avf_bcc_dat",
                 ":test_ref_dt",
             ],
-            enabled: true,
+            // the virtualization stack is not built on the gsi
+            enabled: false,
         },
     },
     out: [
@@ -27,6 +28,8 @@ cc_genrule {
 prebuilt_etc {
     name: "pvmfw_test_img.img",
     filename: "pvmfw_test_img.img",
+    // the virtualization stack is not built on the gsi
+    enabled: false,
     relative_install_path: "pvmfw",
     vendor: true,
     target: {
-- 
2.55.0

